Display issue with flow chart on user-guide/concepts/installing-with-conda page (#13377)
conda/conda: docs/source/user-guide/concepts/installing-with-conda.rst

Installing with conda
=====================

.. _installing-with-conda:

Conda packages can be installed by running the following command::

>   conda install <package>

Reviewer: We should attempt to have a standard for placeholders in commands going forward. I'm not married to any one thing, so `<all-lowercase-words>` works for me. I've also used all-caps, e.g. `<PACKAGE>`.

Any opinions?

Author: I like `<all-lowercase>`. I always get a little anxious when I SEE ALL CAPS!!! 🙀.

Reviewer 2: 🧢🧢🧢

Reviewer: Works for me!

When conda installs a package, it is automatically added to your active
environment. These packages are collections of files and directories
that make up everything you need to use that particular
library or software. For Python packages, these are primarily Python

> files that can be imported in other Python applications, but for compiled

Reviewer: Imported into?

> files that can be imported in other Python applications, but for compiled

Author: ```suggestion
files that can be imported into other Python applications, but for compiled
```

> software packages such as ``ffmpeg`` these are typically binary executables

Reviewer: ```suggestion
software packages, such as ``ffmpeg``, these are typically binary executables
```

you use directly on your computer.

.. admonition:: Note

    If you would like to learn more about how environments are structured,
    head over to :doc:`conda environments<../concepts/environments>`.

> Below is more precise overview of everything that happens during the installation

Reviewer 3: ```suggestion
Below is a more precise overview of everything that happens during the installation
```

Reviewer: Good catch!

process for a single package:

* Currently configured channels (e.g. ``defaults`` or ``conda-forge``) are read in order of priority

> * Repodata for these configured channels will be downloaded and read
> * The repodata will be searched for the package starting with the highest priority channel first
> * Once the package is found, conda makes a separate download request and then installs it
> * This process will then repeat for each of the packages dependencies if there are any

Reviewer: ```suggestion
* Repodata for these configured channels is downloaded and read
* The repodata is searched for the package, starting with the highest priority channel first
* Once the package is found, conda makes a separate download request and then installs it
* This process then repeats for each of the package's dependencies, if there are any
```
Suggestions for keeping the verb tense the same throughout, plus some punctuation edits
